### ISSUE TITLE
copy boot image stuck on fpga

### ISSUE THREAD

**Issue by hhhsiang** [openhwgroup/cva6#1477]
### Is there an existing CVA6 bug for this?

- [X] I have searched the existing bug issues

### Bug Description

I utilize the "make fpga" command to generate .mcs file and subsequently flash it onto the FPGA (Genesys2) using Vivado 2018.2. However, upon completion of the flashing process and subsequent reboot of the FPGA board, it becomes unresponsive, displaying the message "copying boot image done!" as shown in the accompanying images:
![image](https://github.com/openhwgroup/cva6/assets/125340710/6a6c8daa-89d4-4509-b253-e8db41bea25f)
![image](https://github.com/openhwgroup/cva6/assets/125340710/a5aaab91-1aba-411a-8353-73cd78cda7dc)

I have also employed the "make fpga" command with an older version of CVA6-Ariane 4.2, and it worked successfully:
![image](https://github.com/openhwgroup/cva6/assets/125340710/cc6ad40f-c769-49cb-a539-b8c5367e04ae)

How could I fixed it? Thanks in advance!

**Comment by JeanRochCoulon** [comment]
Hello @zchamski    I see you have pushed something to fix this PR. What is the status, does it fix the issue ?

**Comment by zchamski** [comment]
This task should be assigned to @Gchauvon as I'm not involved with the FPGA (except for Spike build interactions).

**Comment by cathales** [comment]
Hello @hhhsiang

Could you please tell us exactly which commit worked and which didn’t so that we can investigate?

This issue is maybe related to #1485

**Comment by hhhsiang** [comment]
@Moschn ,
Yes, I followed the steps. I think the SD card should be ok because CVA6-Ariane 4.2 can correctly work on fpga, as following:
![image](https://github.com/openhwgroup/cva6/assets/125340710/7da33fb7-a5b9-4785-a4ce-3a2ea47692a2)
![image](https://github.com/openhwgroup/cva6/assets/125340710/ed594404-b728-40aa-9c59-32ba99eae810)
![image](https://github.com/openhwgroup/cva6/assets/125340710/2674b7c6-c0f6-4bce-89c9-bddcb9e1eb10)
![image](https://github.com/openhwgroup/cva6/assets/125340710/cbcb1b19-70af-44dd-9d88-cf6947bf7cc1)

**Comment by Moschn** [comment]
Hmm ok. Still very confusing to me that the GPT partitions print all zeros. It was not like this in the past. But maybe something changed in the bootloader in the mean time.

**Comment by hhhsiang** [comment]
@Moschn, I'm uncertain about this. If the SD card hasn't been correctly set up, it shouldn't display the information I mentioned in my previous comment, right?

**Comment by hhhsiang** [comment]
@cathales ,I did not try other commit. I just use `make fpga` command, and flash on to FPGA board. It can work correcly with CVA6-Ariane 4.2, but not with the latest version.

**Comment by cainria** [comment]
Can we close this issue? Otherwise, could https://github.com/openhwgroup/cva6/issues/2334 help?
